### PR TITLE
Led control example

### PR DESCRIPTION
**Pull request by karsev**
This application demonstrates a simple light control application, where a device can control the LED status of all devices in the network. Both Thread and 6LoWPAN can be used.

### CODE REVIEW

**Comment by karsev** [comment]
@mikter @SeppoTakalo

File: mesh_led_control_example.cpp [hunk 1]
```diff
@@ -103,6 +110,7 @@ static void receive() {
         int length = my_socket->recvfrom(&source_addr, receive_buffer, sizeof(receive_buffer) - 1);
         if (length > 0) {
             tr_debug("Packet from %s\n", source_addr.get_ip_address());
+            messageTimeout.attach(&messageTimeoutCallback, MESSAGE_WAIT_TIMEOUT);
```
Comment (mikter) [inline mesh_led_control_example.cpp]: random distribution is needed so that not all devices send the message simultaneously

**Comment by karsev** [comment]
Can this be merged now?

**Comment by 0xc0170** [comment]
How was this tested? We are getting CI failure for this example: 

```
mbed-os-example-mesh-minimal\mesh_led_control_example.cpp",37  Error[Pe020]: identifier "MBED_CONF_APP_BUTTON" is undefined
```

See please https://github.com/ARMmbed/mbed-os/pull/4403#issuecomment-305156239

**Comment by 0xc0170** [comment]
Not certain what is wrong (might be that the example suddenly changed the behaviour and our example test script requires update to reflect  this? As there are app json files here that set LED config but our CI fails to find it), but it's currently blocking any patch to mbed OS ! If this is not resolved soon, I would propose to revert this change-set and we figure out how to fix it if this is not time critical for now.

**Comment by karsev** [comment]
Sorry, I forgot to update the root .json. Created a new PR ->
https://github.com/ARMmbed/mbed-os-example-mesh-minimal/pull/82

**Comment by karsev** [comment]
Accidentally deleted PR 82. 
Now fixed in -> https://github.com/ARMmbed/mbed-os-example-mesh-minimal/pull/83

**Pull request merged**